### DISCUSSION TITLE
Freez of multi option dropdowns

### DISCUSSION THREAD

**Discussion by zssvaidar** [atrocore/atrocore#778] [Q&A] [answered]
![image](https://github.com/atrocore/atrocore/assets/30953478/1a79548f-763a-4f6e-a288-7a333b3387b2)
this commented line could cause screen freez and 100cpu load when list got several 10k options

**Comment by rratsun** [comment, marked as answer]
We have refactored extensible enums and extensible multi enums in 1.6.4. Please, wait when this release become stable.
